From 2f134b84445d2d0bfe3d81276bc7f6648df062ee Mon Sep 17 00:00:00 2001
From: Zhao Yakui <yakui.zhao@intel.com>
Date: Wed, 30 Sep 2009 07:31:57 +0000
Subject: Return failure when the enum-type output property values are not found

Failure to do so causes xrandr to report incorrect property values.

Signed-off-by: Zhao Yakui <yakui.zhao@intel.com>
Signed-off-by: Eric Anholt <eric@anholt.net>
---
diff --git a/src/drmmode_display.c b/src/drmmode_display.c
index 472bc74..3417cab 100644
--- a/src/drmmode_display.c
+++ b/src/drmmode_display.c
@@ -1148,6 +1148,7 @@ drmmode_output_set_property(xf86OutputPtr output, Atom property,
 		    return TRUE;
 		}
 	    }
+	    return FALSE;
 	}
     }
 
--
cgit v0.8.3-6-g21f6
